### PR TITLE
Convert step-14 from using eps/gnuplot to svg/vtu.

### PR DESCRIPTION
**Pull request by bangerth**
The figures as currently used are actually quite ugly. We can do
substantially better.

I've already put the figures on the website.

In reference to #5982.

### CODE REVIEW

File: examples/step-14/doc/results.dox [hunk 1]
```diff
@@ -47,56 +47,62 @@ Refinement cycle: 8
...
+home):
 <table align="center">
   <tr>
     <td width="50%">
-      <img src="https://www.dealii.org/images/steps/developer/step-14.point-value.solution-5.png" alt="">
+      <img src="https://www.dealii.org/images/steps/developer/step-14.point-value.solution-7.9.2.png" alt="">
     </td>
     <td width="50%">
-      <img src="https://www.dealii.org/images/steps/developer/step-14.point-value.solution-5-dual.png" alt="">
+      <img src="https://www.dealii.org/images/steps/developer/step-14.point-value.solution-7-dual.9.2.png" alt="">
```
Comment (drwells) [inline examples/step-14/doc/results.dox]: its not immediately related to this issue, but should we version images in this way? We are currently pretty inconsistent.

**Comment by bangerth** [comment]
On 2/7/20 1:49 PM, David Wells wrote:
> its not immediately related to this issue, but should we version images 
> in this way? We are currently pretty inconsistent.

That's what I've been using for other pictures in this directory. I 
think it's not terribly important, but I'm willing to listen to other 
suggestions (for a follow-up PR if you want).

File: examples/step-14/doc/results.dox [hunk 2]
```diff
@@ -47,56 +47,62 @@ Refinement cycle: 8
...
 refinement criterion involving the dual solution chose to refine them
-differently.
+differently -- because we said that we really only care about a single
+function value somewhere at the top right.
 
 
 
 Looking at the grids that are produced in the course of subsequent
-refinement, here are some of them:
+refinement, here are some of them (in refinement cycles 0, 2, 4 in
```
Comment (GrahamBenHarper) [inline examples/step-14/doc/results.dox]: I think the original sentence actually doesn't sound right when I read it. What about "Here are some of the grids that are produced in the course of subsequent refinement"

File: examples/step-14/doc/results.dox [hunk 3]
```diff
@@ -227,28 +233,33 @@ evaluation shows this:
...
+    <td><img src="https://www.dealii.org/images/steps/developer/step-14.point-derivative.grid-7.9.2.png" alt="" width="100%"></td>
+    <td><img src="https://www.dealii.org/images/steps/developer/step-14.point-derivative.grid-8.9.2.png" alt="" width="100%"></td>
+    <td><img src="https://www.dealii.org/images/steps/developer/step-14.point-derivative.grid-9.9.2.png" alt="" width="100%"></td>
   </tr>
 </table>
 
 Note the asymmetry of the grids compared with those we obtained for
-the point evaluation, which is due to the directionality of the
-x-derivative for which we tailored the refinement criterion.
+the point evaluation. This is due to face that the domain and the primal
```
Comment (GrahamBenHarper) [inline examples/step-14/doc/results.dox]: Should this be "due to the fact"?

**Comment by bangerth** [comment]
Thanks, @GrahamBenHarper! I've fixed the places you mentioned.

**Comment by bangerth** [comment]
Ping?

**Comment by bangerth** [comment]
Oh, so much better: https://dealii.org/developer/doxygen/deal.II/step_14.html#Results

**Pull request merged**